verbal/20160422/main.tex

```diff
@@ -0,0 +1,87 @@
+\documentclass[11pt]{meetingmins}
+\usepackage[utf8]{inputenc}
+
+\setcommittee{BugBusters}
+
+\setmembers{
+
+  D.~Polonio,
+  D.~Rigoni,
+  M.~Di Pirro,
+  G.~Mazzocchin,
+  L.~Bianco,
+  A.~Mantovani,
+  E.~Carraro
+}
+
+\setdate{April 22, 2016}
+
+\setpresent{
+  
+  D.~Polonio,
+  L.~Bianco,
+  M.~Di Pirro,
+  E.~Carraro,
+  D.~Rigoni
+}
+
+\absent{
+
+  G.~Mazzocchin,
+  A.~Mantovani
+}
+
+\alsopresent{None}
+
+\begin{document}
+
+\maketitle
+
+\section{Ordine del giorno}
+
+\begin{enumerate}
+
+\item Discussione su come pianificare l'organizzazione del gruppo
+
+\item Discussione su nuova ridenominazione dei verbali
+
+\item Discussione su l'ordine di codifica di MaaS
+
+\end{enumerate}
+
+\section{Orari}
+
+\begin{itemize}
+\item[Inizio]: ore 17:30
+\item[Fine]: ore 18:30
+\item[Tempo]: Previsto di 60 min, effettivo di 60 min
+
+\end{itemize}
+
+\section{Verbale}
+
+\subsection{Discussione su come pianificare l'organizzazione del gruppo}
+
+Si discute su come iniziare a scrivere MaaS, elencando tutti i vantaggi e gli svantaggi di diverse strategie di implementazioni. Inoltre, sono state definite le attività da svolgere e sono stati identificati i principali task.
+
+\subsection{Discussione su nuova ridenominazione dei verbali}
+
+Viene approvato da tutti di rinominare i verbali secondo l'identificativo suggerito in fase di correzione della Revisione di Progettazione. La rinominazione viene stabilito avere effetto retroattivo, per poter uniformare l'archivio di tutti i verbali fin'ora scritti.
+
+\subsection{Discussione su l'ordine di codifica di MaaS}
+
+Si discute su una possibile strategia per la codifica di MaaS, valutando attentamente svantaggi e vantaggi di eventuali scelte strategiche. Alla fine, viene deciso per sviluppare MaaS nel seguente ordine:
+\begin{enumerate}
+
+\item Backend
+\item Frontend
+\item Interprete
+\item Editor grafico
+\end{enumerate}
+
+Con questo ordine di sviluppo sar\`a possibile ridurre al minimo il numero di \textit{stub} e \textit{driver} necessari e consentir\`a al gruppo una maggiore velocit\`a nello sviluppo. L'editor grafico viene tenuto alla fine in quanto essendo facoltativo verr\`a codificato nei tempi di \textit{slack} accomulato per essere terminato, se possibile, nella prossima fase (revisione d'accettazione).
```

Comment:
Okok!:)
Il 28 apr 2016 00:08, "Matteo Di Pirro" <notifications@github.com> ha
scritto:

> In verbal/20160422/main.tex
> <https://github.com/BugBusterSWE/documentation/pull/249#discussion_r61345385>
> :
>
> > +\subsection{Discussione su nuova ridenominazione dei verbali}
> > +
> > +Viene approvato da tutti di rinominare i verbali secondo l'identificativo suggerito in fase di correzione della Revisione di Progettazione. La rinominazione viene stabilito avere effetto retroattivo, per poter uniformare l'archivio di tutti i verbali fin'ora scritti.
> > +
> > +\subsection{Discussione su l'ordine di codifica di MaaS}
> > +
> > +Si discute su una possibile strategia per la codifica di MaaS, valutando attentamente svantaggi e vantaggi di eventuali scelte strategiche. Alla fine, viene deciso per sviluppare MaaS nel seguente ordine:
> > +\begin{enumerate}
> > +
> > +\item Backend
> > +\item Frontend
> > +\item Interprete
> > +\item Editor grafico
> > +\end{enumerate}
> > +
> > +Con questo ordine di sviluppo sar\`a possibile ridurre al minimo il numero di \textit{stub} e \textit{driver} necessari e consentir\`a al gruppo una maggiore velocit\`a nello sviluppo. L'editor grafico viene tenuto alla fine in quanto essendo facoltativo verr\`a codificato nei tempi di \textit{slack} accomulato per essere terminato, se possibile, nella prossima fase (revisione d'accettazione).
>
> Sicuro di voler usare "Fase", non era un errore segnalato durante la RR?
> Togli "se possibile", non mi sembra il caso di partire pensando di non
> finire
> Togli "revisione di accettazione"
> In generale cambierei la frase. Lo codifichiamo più avanti perché abbiamo
> più tempo, non so se si possa definire slack.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/BugBusterSWE/documentation/pull/249/files/e126a9295eb58e1c7a60a52743be56a24e4adcbd#r61345385>
>